src/Components/TopMenu.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {NavLink, useHistory} from "react-router-dom";
+
+function TopMenu({auth, getter, setter}) {
+    const history = useHistory();
+
+    const logOutClick = (e) => {
+        setter(!getter);
```

Comment:
Het is veiliger om hier ipv "!getter" "false" in te zetten. Mocht iemand nu namelijk per ongeluk op deze knop kunnen drukken terwijl diegene niet ingelogd is, dan logt diegene nu met de knop in, omdat je de waarde verandert in het tegenovergestelde.